django/santropolFeast/order/models.py

```diff
@@ -147,8 +146,7 @@ def create_orders_on_defaults(creation_date, delivery_date, clients):
             if free_side_dish_qty == 0:
                 continue  # No meal for client on this day
             try:
-                order = Order.objects.get(
-                    client=client, delivery_date=delivery_date)
+                Order.objects.get(client=client, delivery_date=delivery_date)
                 # order already created, skip order items creation
                 # (if want to replace, must be deleted first)
                 continue
```

Comment:
Hi @lamontfr,
Yes, I'm in Montreal.
You can send me your contact information to my email erozqba@gmail.com and when would be a good time to call you.